designing-event-driven-processes/README.md

```diff
@@ -0,0 +1,22 @@
+# Beyond basic messaging: Designing long-running processes
```

Comment:
```suggestion
# Message Mapping: Designing complex business processes with messaging
```

Reply:
👎  It pictures "Message Mapping" as a thing, but it's not.

Reply:
how about "Designing intention-revealing business processes with messaging"

Reply:
that or just "Designing complex business processes with messaging"